integration-tests/manifests/annotations/validate_annotations_test.go

```diff
@@ -0,0 +1,1003 @@
+package annotations
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/aws/amazon-cloudwatch-agent-operator/pkg/instrumentation/auto"
+	"github.com/stretchr/testify/assert"
+	appsV1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+)
+
+const injectJavaAnnotation = "instrumentation.opentelemetry.io/inject-java"
+const autoAnnotateJavaAnnotation = "cloudwatch.aws.amazon.com/auto-annotate-java"
+const injectPythonAnnotation = "instrumentation.opentelemetry.io/inject-python"
+const autoAnnotatePythonAnnotation = "cloudwatch.aws.amazon.com/auto-annotate-python"
+const deploymentName = "sample-deployment"
+const statefulSetName = "sample-statefulset"
+const amazonCloudwatchNamespace = "amazon-cloudwatch"
+
+const daemonSetName = "sample-daemonset"
+
+const amazonControllerManager = "cloudwatch-controller-manager"
+
+var opMutex sync.Mutex
+
+// ---------------------------USE CASE 1 (Java and Python on Deployment) ----------------------------------------------
+func TestUseCase1(t *testing.T) {
```

Comment:
nit: Better function naming such as `TestJavaAndPythonDeployment`. This helps with `go test` output instead of having to go back and read the comment and remember what the test was for